src/theme/Palette.js

```diff
@@ -7,13 +7,13 @@ const colors = getColors()
 export const day = {
   mode: 'light',
   primary: {
-    main: colors.grey.medium,
+    main: '#6D8752',
```

Comment:
Same here.. I think we should avoid deviation from the palette combos in Material if possible

Reply:
This one is primary green - mui greens are not so nice